apps/services/sessions/test/session.fixture.ts

```diff
@@ -1,15 +1,25 @@
 import faker from 'faker'
 
+import { createNationalId } from '@island.is/testing/fixtures'
+
 import { CreateSessionDto } from '../src/app/sessions/create-session.dto'
 
-export const createRandomSession = (
-  actorNationalId: string,
+export interface CreateSessionDtoOptions {
+  subjectType?: any
```

Comment:
We should reuse the type form the `createNationalId` for this property. Helps with DX when adding tests later on.

Reply:
Fixed.